R/cchart.R

```diff
@@ -308,17 +308,23 @@ addWarning <- function(x, chart.type, small.multiples, has.annotations)
     msg <- ""
 
     if (small.multiples)
-        msg <- "This visualization is a small multiple which is not supported by Microsoft."
+        msg <- "This visualization is a small multiple which is not supported by PowerPoint."
     else if (has.annotations)
-        msg <- "This visualization contains annotations which is not supported by Microsoft."
+        msg <- "This visualization contain annotations which are not supported by PowerPoint."
```

Comment:
Revert `contain` back to `contains`. Sounds odds otherwise.
i.e.
`This visualization contains annotations which are not supported by PowerPoint.`

Reply:
Fixed in 557b7d0